helm/blueapi/templates/deployment.yaml

```diff
@@ -70,6 +70,23 @@ spec:
           env:
             - name: SCRATCH_AREA
               value: {{ .Values.scratch.containerPath }}
+        {{- if .Values.listener.enabled -}}
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
```

Comment:
Resources should be much smaller than the resources for the blueapi main container

Reply:
I'll just remove the resource requirements entirely and let k8s work it out. Do you think that's reasonable?

The alternative is including a listener.resources section in the values.yaml which I don't know if we should do since it's such a small process